deny.toml

```diff
@@ -294,6 +296,12 @@ skip = [
     { name = "uuid", version = "*" },
     { name = "wasi", version = "*" },
     { name = "webpki", version = "*" },
+    { name = "windows_aarch64_msvc", version = "*" },
+    { name = "windows_i686_gnu", version = "*" },
+    { name = "windows_i686_msvc", version = "*" },
+    { name = "windows-sys", version = "*" },
+    { name = "windows_x86_64_msvc", version = "*" },
+    { name = "windows_x86_64_gnu", version = "*" },
```

Comment:
are these needed anymore since we ignored the vul ID already?

Reply:
Not really needed, but they were cluttering the Cargo deny checks CI log

Reply:
if they don't fail the pipeline, let's remove these

I will do more work on vulnerable dependencies this week